section2/exercises/if_statements.rb

```diff
@@ -61,5 +77,15 @@
 # Experiment with manipulating the value held within both variables
 # to make sure all above conditions output what you expect.
 
-cups_of_flour = 1
+cups_of_flour = 3
 has_sauce = true
+
+if cups_of_flour == 1 && has_sauce == true
```

Comment:
Just a challenge since your work is strong:

Could you rewrite this so that there is no redundant code (like lines 84 and 86 - and 88 and 90)?